zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/NewSessionsServiceImpl.java

```diff
@@ -0,0 +1,139 @@
+package com.github.onsdigital.zebedee.session.service;
+
+import com.github.onsdigital.zebedee.reader.util.RequestUtils;
+import com.github.onsdigital.zebedee.session.model.Session;
+import com.github.onsdigital.zebedee.user.model.User;
+import com.session.service.client.SessionClient;
+import com.session.service.client.SessionClientImpl;
+import com.session.service.entities.SessionCreated;
+import com.session.service.error.SessionClientException;
+import org.apache.commons.lang.StringUtils;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.Date;
+
+public class NewSessionsServiceImpl implements Sessions {
```

Comment:
As this is a new class you should write the appropriate Javadoc on the class and the public methods. It doesn't have to be an essay just a succinct description of what this class does + any useful information anyone using it will need to know.